CODEOWNERS

```diff
@@ -1,6 +1,41 @@
 # Use owners-snuba by default
 * @getsentry/owners-snuba
 
+# Datasets
+/snuba/datasets/configuration/discover              @getsentry/discover-n-dashboards
+/snuba/datasets/configuration/events                @getsentry/issues
+/snuba/datasets/configuration/issues                @getsentry/issues
+/snuba/datasets/configuration/group_attributes      @getsentry/issues
+/snuba/datasets/configuration/groupassignee         @getsentry/issues
+/snuba/datasets/configuration/sessions              @getsentry/performance
```

Comment:
This dataset is deprecated, and I think will be removed. Let's just leave it off the list.